packages/ui/src/components/callout/callout.web.tsx

```diff
@@ -31,38 +31,71 @@ const calloutRecipe = cva({
       },
     },
   },
-  defaultVariants: {
-    variant: 'default',
-  },
 });
 
-type CalloutVariants = RecipeVariantProps<typeof calloutRecipe>;
+const icons = {
+  info: <InfoCircleIcon />,
+  success: <CheckmarkCircleIcon />,
+  warning: <ErrorTriangleIcon />,
+  error: <ErrorTriangleIcon />,
+  default: null,
+};
 
-interface CalloutProps extends BoxProps {
-  icon?: ReactNode;
+interface CalloutOwnProps {
+  /** Title of the callout.  */
```

Comment:
I don't think all of these comments are necessary

Reply:
I think that here the only useful comment is:
```
Limit the content width to the specified value. Suitable for full-width callouts displayed at the top of the page.
```

The rest are self explanatory. If you want to add docs I would add them into the `.story` file instead, maybe in the docs description?

```
const meta: Meta<typeof Component> = {
  component: Component,
  tags: ['autodocs'],
  title: 'Callout',
  parameters: {
    docs: {
      description: { 
          component: '...',
      },
```